### PR TITLE
Pulling initial prismic data

### PR DESCRIPTION
**Pull request by jean-zhang**
## Changes
- Pulled in initial Home Page and Story data
- Built out initial structure for a `SliceZone` component and child `TextSection` and `ImageGallery` components

## Images 
- Looking to set up deployment previews next sprint (not sure if needs any tweaks yet)

Screen Shot 2021-02-23 at 6 00 01 PM
Screen Shot 2021-02-23 at 5 58 50 PM

## Notes
- `npm install` needed
- To develop locally take the `env_example` file and rename to `.env` and update the XXXXX with our API URL (found on the prismic)

### CODE REVIEW

**Review by isabellberzsenyi — APPROVED** [review]
Looks great to me!

**Review by jzhzhang66 — APPROVED** [review]
Also looks good to me!

File: pages/index.js [hunk 1]
```diff
@@ -1,65 +1,31 @@
...
-          rel="noopener noreferrer"
-        >
-          Powered by{' '}
-          <img src="/vercel.svg" alt="Vercel Logo" className={styles.logo} />
-        </a>
-      </footer>
-    </div>
-  )
+export async function getStaticProps() {
+  const home = await client.getSingle("home_page");
```
Comment (josephmannis) [inline pages/index.js]: So here is where I would abstract out the prismic client. You could have a dedicated js (something like api/content-service.js) file that exports a couple of async functions for getting content/pages/etc. Then have this function call the exposed one. This keeps your pages away from the fact that you're using prismic and you can easily centralize/reuse all of your fetching logic.

File: pages/story/[uid].js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import SliceZone from "../../components/slices/SliceZone";
+
+export default function Story({ data }) {
+  return (
+    <article>
+      <header>{RichText.asText(data.title)}</header>
+      <SliceZone sliceZone={data.body} />
+    </article>
+  );
+}
```
Comment (josephmannis) [inline pages/story/[uid].js:19]: I also recommend using PropTypes

File: components/slices/TextSection.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const TextSection = ({ slice }) => {
+  return (
+    <>
+      <h3>Text Section: </h3>
+      <p>{slice.primary.text[0].text}</p>
```
Comment (josephmannis) [inline components/slices/TextSection.js]: Recommendation: 
- For text, use the prismic client rich text renderer from the start (though I see you are doing this already elsewhere). I recommend making your own RichText component that wraps the Prismic one to abstract away implementation details. 
- Keep the `slice.primary.text...` stuff out of the presentational component. You can format the props to be more friendly in the `getStaticProps` function. This just further abstracts away Prismic and future-proofs your site. You could define some more friendly props for TextSection and have SliceZone pass those in.

**Review by josephmannis — CHANGES_REQUESTED** [review]

File: pages/story/[uid].js (hunk 2 above)
Comment (jean-zhang) [reply pages/story/[uid].js:19]: Discussed offline: looking to add PropTypes once we've stabilized on prismic data.

**Review by isabellberzsenyi — APPROVED** [review]
looks good to me

**Pull request merged**